webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path')
+// const webpack = require('webpack')
+
+module.exports = {
+  entry: [
+    './lib/index.js'
```

Comment:
Why does webpack depend on the compiled `lib`? Since [`babel` is ran along webpack](https://github.com/linkeddata/rdflib.js/pull/199/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR18) anyway, we might want to run webpack on `src` and do additional magic there when needed for the browser.

Drawback: slower compilation time. Advantage: not having to run `npm run lib` first (which we probably want to only run on distribution anyway).